provider/core/src/data_provider.rs

```diff
@@ -141,15 +141,25 @@ where
 impl<'d, 's, M> Clone for DataPayload<'d, 's, M>
 where
     M: DataMarker<'s>,
+    for<'a> <M::Yokeable as Yokeable<'a>>::Output: Clone,
```

Comment:
Let's mark this (and the impl on DataResponse) with a comment linking to the rustc bug so folks don't get confused when this _doesn't_ work

Reply:
Added comment.